src/util/resolveRedirectTo.js

```diff
@@ -11,6 +11,9 @@ export default (redirectTo, basePath, id) => {
         case 'show':
             return `${linkToRecord(basePath, id)}/show`;
         default:
+            if (typeof redirectTo === 'function') {
```

Comment:
Please add a unit test for this case in `resolveRedirectTo.spec.js`.
Pass a function as the `redirecTo` which returns its parameters and asserts they are corrects